pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -300,6 +300,13 @@ self: super: ({
     __darwinAllowLocalNetworking = true;
   });
 
+  # network requires `IP_RECVTOS`, which was added in 10.15.
```

Comment:
It was added in 10.13!

Reply:
```
$ nix build github:nixos/nixpkgs/nixos-24.11#legacyPackages.x86_64-darwin.apple-sdk_10_13
$ rg "IP_RECVTOS" result
result/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk/System/Library/Frameworks/Kernel.framework/Versions/A/Headers/netinet/in.h
431:#define     IP_RECVTOS              27   /* bool; receive IP TOS w/dgram */

result/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk/usr/include/netinet/in.h
433:#define     IP_RECVTOS              27   /* bool; receive IP TOS w/dgram */

Reply:
This is gone in 25.05 anyway as we dropped support for versions below macOS 11. It also shouldn't matter unless the package also uses 10.14+-only APIs.